tests/src/JIT/Directed/arglist/vararg.cs

```diff
@@ -0,0 +1,1195 @@
+            3L,
+            80L,
+            7L,
+            -1L,
+            48L,
+            66L,
+            23L,
+            62L,
+            1092L,
+            -890L,
+            -20L,
+            -41L,
+            88L,
+            98L,
+            1L,
+            2L,
+            3L,
+            4012L,
+            16L,
+            673L,
+            873L,
+            45L,
+            85L,
+            -3041L,
+            22L,
+            62L,
+            401L,
+            901L,
+            501L,
+            1001L,
+            1002L
+        }) ? success : 6;
+
+        success = TestPassingManyFloats(new float[]
+        {
+            1002,
+            40,
+            39,
+            12,
+            14,
+            -502,
+            -13,
+            11,
+            98,
+            45,
+            3,
+            80,
+            7,
+            -1,
+            48,
+            66,
+            23,
+            62,
+            1092,
+            -890,
+            -20,
+            -41,
+            88,
+            98,
+            1,
+            2,
+            3,
+            4012,
+            16,
+            673,
+            873,
+            45,
+            85,
+            -3041,
+            22,
+            62,
+            401,
+            901,
+            501,
+            1001,
+            1002
+        }) ? success : 7;
+
+        success = TestPassingManyDoubles(new double[]
+        {
+            1002,
+            40,
+            39,
+            12,
+            14,
+            -502,
+            -13,
+            11,
+            98,
+            45,
+            3,
+            80,
+            7,
+            -1,
+            48,
+            66,
+            23,
+            62,
+            1092,
+            -890,
+            -20,
+            -41,
+            88,
+            98,
+            1,
+            2,
+            3,
+            4012,
+            16,
+            673,
+            873,
+            45,
+            85,
+            -3041,
+            22,
+            62,
+            401,
+            901,
+            501,
+            1001,
+            1002
+        }) ? success : 8;
+
+        success = TestPassingIntsAndLongs(new int[] { 100, 200 }, new long[] { 102312131L, 91239191L }) ? success : 9;
+        success = TestPassingFloatsAndDoubles(new float[] { 100.0F, 200.0F }, new double[] { 12.1231321, 441.2332132335342321 }) ? success : 10;
+
+        // Try passing empty varargs.
+        success = TestPassingEmptyInts(new int[] { }) ? success : 11;
+        success = TestPassingEmptyLongs(new long[] { }) ? success : 12;
+        success = TestPassingEmptyFloats(new float[] { }) ? success : 13;
+        success = TestPassingEmptyDouble(new double[] { }) ? success : 14;
+
+        //success = TestPrintF() ? success : 15;
+
+        int returnValue = TestPassingStructs();
+        success = returnValue == 100 ? success : returnValue;
```

Comment:
One downside of this test methodology is that you learn the first test that failed (assuming all the failure modes have unique error values), but if there is more than one failure, you don't learn about it since there is no error output. Despite that, you still run all the tests even though we don't learn if they pass or fail.